test/runnable/gdb15729.sh

```diff
@@ -1,24 +1,17 @@
 #!/usr/bin/env bash
 
-src=runnable${SEP}extra-files
-dir=${RESULTS_DIR}${SEP}runnable
-output_file=${dir}/gdb15729.sh.out
 
-libname=${dir}${SEP}lib15729${LIBEXT}
-
-$DMD -g -m${MODEL} -I${src} -of${libname} -lib ${src}${SEP}lib15729.d || exit 1
-$DMD -g -m${MODEL} -I${src} -of${dir}${SEP}gdb15729${EXE} ${src}${SEP}gdb15729.d ${libname} || exit 1
+$DMD -g -m${MODEL} -I${EXTRA_FILES} -of${RESULTS_OUTPUT}${LIBEXT} -lib ${EXTRA_FILES}${SEP}lib15729.d || exit 1
+$DMD -g -m${MODEL} -I${EXTRA_FILES} -of${RESULTS_OUTPUT}${EXE} ${EXTRA_FILES}${SEP}gdb15729.d ${RESULTS_OUTPUT}${LIBEXT} || exit 1
```

Comment:
All these `|| exit 1` should be removed to and be replaced with `set -e`, but that's a task for another PR.